server/src/main/webapp/WEB-INF/rails/webpack/views/pages/pipeline_activity/comment_widget.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {MithrilViewComponent} from "jsx/mithril-component";
+import _ from "lodash";
+import m from "mithril";
+import Stream from "mithril/stream";
+import {Secondary} from "../../components/buttons";
```

Comment:
Suggestion: We prefer absolute import path `views/components/...` over relative one. Please update this.

Reply:
Done